components/desktop/gnome2/gnome-connection-manager/Makefile

```diff
@@ -10,34 +10,55 @@
 #
 
 #
-# Copyright (c) 2015 Jim Klimov
+# Copyright (c) 2015-2020 Jim Klimov
 #
 
 include ../../../../make-rules/shared-macros.mk
 
-COMPONENT_NAME=		gnome-connection-manager
+COMPONENT_NAME_LONG=	gnome-connection-manager
```

Comment:
Why do you separate COMPONENT_NAME and COMPONENT_NAME_LONG?

Reply:
This simplified building the alternate "gcm" named package for tests by just flipping a switch in a copy of the makefile. In the original project and its different hosting variants, the two strings are also variably used adding to the mess.